src/libAtomVM/bif.c

```diff
@@ -197,6 +197,18 @@ term bif_erlang_is_tuple_1(Context *ctx, uint32_t fail_label, term arg1)
     return term_is_tuple(arg1) ? TRUE_ATOM : FALSE_ATOM;
 }
 
+term bif_erlang_is_record_2(Context *ctx, uint32_t fail_label, term arg1, term arg2)
+{
+    UNUSED(ctx);
+    if (!term_is_tuple(arg1) || term_get_tuple_arity(arg1) == 0) {
```

Comment:
Sorry, I cannot merge this yet, we are not 1:1 with OTP.
arg2 validation should happen first, look here:
```erl
1> erlang:is_record({}, 5).
** exception error: bad argument
     in function  is_record/2
        called as is_record({},5)
        *** argument 1: not an atom
```

We should add this to our tests.